docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -254,29 +318,35 @@ To uninstall the agent, follow the instructions specific to your OS.
 
 The agent offers to send logs contained in files to Sekoia.io.
 
-If you want to enable this feature, follow these steps: 
+If you want to enable this feature, follow these steps:
 
 1. Edit the configuration file at:
-	
-	=== "Windows"
-	
-	    ```
-	    C:\Windows\System32\config\systemprofile\AppData\Local\Sekoia.io\EndpointAgent\config.yaml
-	    ```
-	
-	=== "Linux"
-	
-	    ```
-	    /etc/endpoint-agent/config.yaml
-	    ```
```

Comment:
Should be kept idented or messes up the bullet numbers:

![image](https://github.com/SEKOIA-IO/documentation/assets/679995/7ff74c35-2a56-4246-8491-528689a51d6b)